runner/app/live/trickle/trickle_subscriber.py

```diff
@@ -62,14 +51,19 @@ async def next(self):
             self.pending_get = None
 
             # Extract and set the next index from the response headers
-            idx = await self.get_index(conn)
-            if idx != -1:
+            segment = Segment(conn)
```

Comment:
Is this `conn` var actually a response obj? Got slightly confused that it's passed as `conn` and becomes `resp(onse)` on these functions (old `get_index` or `Segment` constructor). Might be worth renaming here

Reply:
They are the same, renamed to `resp` for consistency in https://github.com/livepeer/ai-worker/pull/271/commits/8b411dc94000f71d3e4bbcafa9224a8b782dd00a